packages/api/src/schema/api-schema.yaml

```diff
@@ -2488,22 +3097,572 @@ paths:
+          required: true
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/signing-key"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    patch:
+      tags:
+        - Signing Key
+      summary: Update a signing key
+      description: >
+        PATCH /access-control/signing-key/{KEY_ID} updates an existing signing
+        key. It is a single endpoint that can update any of the mutable
+        properties of a stream. Specifically:
+
+
+        - name
+
+        - disabled
+
+        All fields are optional, and if any field is not included in the request
+        payload it will be kept unchanged from the saved object.
+      parameters:
+        - in: path
+          name: keyId
+          schema:
+            type: string
+          description: ID of the signing key
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                disabled:
+                  type: boolean
+                name:
+                  type: string
+              additionalProperties: false
+      responses:
+        "204":
+          description: Success (No content)
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  /task:
+    get:
+      tags:
+        - Task
+      summary: Retrieve Tasks
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/task"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  "/task/{taskId}":
+    get:
+      tags:
+        - Task
+      summary: Retrieve a Task
+      description: >-
+        Retrieves the details of a Task, including progress updates and status.
+        Supply the unique Task ID that was returned from your previous request
+        or from an Asset import or export.
+      parameters:
+        - in: path
+          name: taskId
+          schema:
+            type: string
+          description: ID of the task
+          required: true
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/task"
+        default:
+          description: Error
+  /transcode:
+    post:
+      tags:
+        - Misc
+      summary: Transcode a video
+      description: >
```

Comment:
Omg this is so long. We need a guide for this as well (not a change for this PR)